packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/BarPlotMarkerConfigurationMenu.tsx

```diff
@@ -26,16 +37,73 @@ interface Props
   > {
   onChange: (configuration: BarPlotMarkerConfiguration) => void;
   configuration: BarPlotMarkerConfiguration;
+  overlayConfiguration: OverlayConfig | undefined;
+  overlayVariable: Variable | undefined;
+  subsettingClient: SubsettingClient;
+  studyId: string;
+  filters: Filter[] | undefined;
 }
 
+// TODO: generalize this and PieMarkerConfigMenu into MarkerConfigurationMenu. Lots of code repitition...
```

Comment:
https://blog.codinghorror.com/rule-of-three/

Reply:
Perhaps I'll change it to a "maybe do" 😆

Reply:
🙃  
Just wanted to derp by the water color 👋